DRAFT.md

```diff
@@ -715,6 +715,24 @@ constexpr indirect<T, Alloc>& operator->() noexcept;
 
 Otherwise, the interface of the specialization is as defined in [optional].
 
+#### X.Y.13 Formatter support [indirect.fmt]
+
+```c++
+// [indirect.fmt]
+template <class T, class Alloc, class charT>
+  requires { std::formatter<T, char>{}; };
+struct std::formatter<indirect<T, Alloc>, charT> : std::formatter<T> {
+  constexpr typename FormatContext::iterator parse(format_parse_context& ctx);
+
+  template<class FormatContext>
+  typename FormatContext::iterator format(
+    indirect<T, Alloc> const& value, FormatContext& ctx) const;
+};
+```
+
+Specialization of `std::formatter<indirect<T, Alloc>, charT>` when the underlying
```

Comment:
@jbcoe let me know if the wording can be improved here.

Reply:
Maybe “T supports” (missing s) but otherwise looks great.

Reply:
Ok, added thanks.